lib/beaker-hostgenerator/data.rb

```diff
@@ -717,22 +508,6 @@ def osinfo
             'template' => 'palo-alto-8.1.0-x86_64',
           },
         },
-        'opensuse11-32' => {
-          :general => {
-            'platform' => 'opensuse-11-i386',
-          },
-          :vmpooler => {
-            'template' => 'opensuse-11-i386',
-          },
-        },
-        'opensuse11-64' => {
-          :general => {
-            'platform' => 'opensuse-11-x86_64',
-          },
-          :vmpooler => {
-            'template' => 'opensuse-11-x86_64',
-          },
-        },
```

Comment:
I think this is fine to remove, because I don't think we test against opensuse and we were only testing against SLES 11 in Puppet 6.x which is now EOL. Here is the corresponding removal ticket https://tickets.puppetlabs.com/browse/PA-5104